projects/plugins/crm/includes/ZeroBSCRM.MetaBoxes3.Contacts.php

```diff
@@ -271,7 +271,7 @@ public function html( $contact, $metabox ) {
                     # ... further hacked
 
                     if ($zbsShowID == "1" && isset($contact['id']) && !empty($contact['id'])) { ?>
-                    <tr class="wh-large"><th><label><?php esc_html_e('Customer',"zero-bs-crm");?> ID:</label></th>
+							<tr class="wh-large"><th><label><?php esc_html_e( 'Contact', 'zero-bs-crm' ); ?> ID:</label></th>
```

Comment:
Should we perhaps improve the translatability of this one since we're already touching it? 😄
The current way we implement it assumes that it's called `ID` in all languages:

```suggestion
							<tr class="wh-large"><th><label><?php esc_html_e( 'Contact ID', 'zero-bs-crm' ); ?>:</label></th>
```